browser/file_persister_test.go

```diff
@@ -0,0 +1,173 @@
+package browser
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/grafana/xk6-browser/env"
+	"github.com/grafana/xk6-browser/storage"
+)
+
+func Test_newFilePersister(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name      string
+		envLookup env.LookupFunc
+		wantType  filePersister
+		wantErr   bool
+	}{
+		{
+			name: "local_no_env_var",
+			envLookup: func(key string) (string, bool) {
+				return "", false
+			},
+			wantType: &storage.LocalFilePersister{},
+		},
+		{
+			name: "local_empty_env_var",
+			envLookup: func(key string) (string, bool) {
+				return "", true
+			},
+			wantType: &storage.LocalFilePersister{},
+		},
+		{
+			name: "remote",
+			envLookup: func(key string) (string, bool) {
+				return "url=https://127.0.0.1/,basePath=/screenshots,header.1=a", true
+			},
```

Comment:
Using the environment variable key makes it more explicit.

```suggestion
			envLookup: ConstLookup(env.ScreenshotsOutput, "url=https://127.0.0.1/,basePath=/screenshots,header.1=a")
```

Reply:
Fixed in 54d5961592d1531867598ba3b75bd5e1ad392cb0